list.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    yield @list
+  end
+end
+list1 = MyList.new(1, 2, 3, 4)
+
+# Test #all?
+puts(list1.all? { |e| e < 5 })
+puts(list.all? { |e| e > 5 })
+
+# Test #any?
+puts(list1.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+
+# Test #filter
+puts list1.filter(&:even?)
```

Comment:
- [ ] The Result for the 4th test is not expected, The result should be `false` but it's `[1, 2, 3, 4]`. Please fix this issue.

### Your result

![image](https://user-images.githubusercontent.com/83803414/209147464-899e66b4-ed7f-4f6c-83d9-8b8fe9ca194a.png)

### Expected result

![image](https://user-images.githubusercontent.com/83803414/209147529-f98db503-3691-49d0-a96c-9b3e00fc7e19.png)

Reply:
Thanks. Its now resolved